setup.py

```diff
@@ -1,12 +1,17 @@
-from setuptools import setup, find_packages, Extension
-from setuptools.command.build_ext import build_ext as _build_ext
+import multiprocessing
+import os
 import sys
+from pathlib import Path
+
 import numpy
-import os
-import os.path as path
-import multiprocessing
+from setuptools import Extension, find_packages, setup
+from setuptools.command.build_ext import build_ext as _build_ext
 
-multiprocessing.set_start_method('fork')
+
+if os.name == "nt":
```

Comment:
Raysect 0.8.x does not support Windows. The multiprocessing paradigm it uses relies on the Linux kernel's copy-on-write when forking processes, and this is not something Windows supports. The fix is to use a new parallel render engine which doesn't require `fork`'s COW semantics, and this is not a small project. Adjusting setup.py to support building on Windows may give Windows users the false impression that they should expect Raysect to work well on this platform.

Reply:
Thank you for your review!
Is the incompatibility for Windows tied to all multiprocessing methods? If so, can I also fix this issue by modifying the `workflow.py` module?

Reply:
Changing "fork" to "spawn" in `MulticoreEngine` will cause local copies of the entire scene graph, including all its data, to be created in child processes. Meshes used in Raysect/Cherab scenes can take up several gigabytes. With dozens of parallel processes, this can lead to out of memory problem, not to mention possible performance issues. The scene graph must be shared between all the processes. This is currently achieved using the POSIX `fork()` Copy-on-Write semantics. I don't know of any other way to do this in pure Python. I think the best way to use Raysect on Windows machines is via [WSL2](https://learn.microsoft.com/en-us/windows/wsl/install).

Reply:
Python 3.13 added an experimental [free-threading build](https://docs.python.org/3/howto/free-threading-python.html). In the future, this may replace multiprocessing in Raysect's `MulticoreEngine`.

Reply:
If Matt and I ever get time to return to Raysect development (its not impossible, just impractical for now) I'll like to experiment with the new threading model.

Reply:
@vsnever, thank you for your comment! "fork" seems to use a shared memory scheme, doesn't it?
The GIL-free threading is very attractive and could be an option in the future.

@CnlPepper Thank you for commenting! I can't wait for you to come back as a developer😢

Reply:
I changed the "spawn" setting in the `if` case to `raise OSError` to throw an error if the OS is Windows.